docs/docs/adding-search-with-algolia.md

```diff
@@ -53,13 +57,21 @@ module.exports = {
 
 Notice that you're loading `queries` from a file at `./src/utils/algolia.js` (you can of course put it wherever you like) and your Algolia ID and API key from `.env` so let's add those files.
 
+For this, you will need to navigate to [the 'API Keys' section of your Algolia profile](https://www.algolia.com/api-keys). If you already have an account, you will find your API keys here. If not, you will need to sign up for one and then navigate to this link. It should look something like this screenshot:
+
+![algolia api key screenshot](./images/algolia-api-keys.png)
```

Comment:
The screenshot is really tall, and cuts off the screen–I'm wondering if a wider screenshot that shows more of the page, perhaps with an arrow pointing to where the API keys could be found would help people (and fit better in the doc layout). I'd recommend `rebeccapurple` (`#663399`) for an arrow color if you end up going that route.

Reply:
Good call, I'll trim it down a bit and try to be more illustrative. On second look, it's definitely difficult to see what's going on at that page.